### ISSUE TITLE
🐛 Bug: jstz install: incorrect error reporting on readonly shell configuration file

### ISSUE THREAD

**Issue by emturner** [trilitech/jstz#471]
# Description

My system has a readonly `~/.bashrc`. The installer (understandably) fails to add the alias, but reports it as having succeeded.

### Observed behaviour

It claim the alias was successfully added, and then messes up my shell prompt (running in `zsh`, but my user `$SHELL` is bash.

```
Pulling jstz-cli docker container from GHCR...
20240313: Pulling from trilitech/jstz-cli
Digest: sha256:f320f5a2247acb08190db13429d600eef45ad29c03a76fbf47685183ecb1ef34
Status: Image is up to date for ghcr.io/trilitech/jstz-cli:20240313
ghcr.io/trilitech/jstz-cli:20240313

Configuring jstz...Configuration unchanged. No update needed.
Configuring `jstz` alias...
jstz_configure:64: permission denied: /home/emma/.zshrc
Alias added to /home/emma/.zshrc.
```

### Expected behaviour

The installer should either fail, or perhaps output a request for me to add an alias (that I could add manually).

### Steps to reproduce

From within zsh:
```
chmod u=r,og=r ~/.zshrc
source <(curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/trilitech/jstz/main/scripts/install-jstz-cli.sh)
```

**Comment by emturner** [comment]
Hey! 

If you want to determine the currently running shell, a more consistent way is to use `$0`. While this doesn't work inside a script, we're sourcing the script when installing - so actually it works well in this instance.

I got it working for me - ended up adding the alias manually (which I would expect to be told to do anyway, post the patches).

Happy to retest once it's updated :) 

Thanks!

**Comment by emturner** [comment]
fixed by #474
